README.php

```diff
@@ -29,4 +29,6 @@
  *
  * Now am changing commit again before push
  *
+ * Now changing in new branch hotfix/CR143
```

Comment:
Requested changes are made.. Now approve the pull request and merge to master branch